### PR TITLE
refactor: get selected wallet and network by default

### PR DESCRIPTION
**Pull request by ovitrif**
### Description

Refactors the code to replace occurrences of
```js
if (!selectedNetwork) {
  selectedNetwork = getSelectedNetwork();
}
```
and / or :
```js
if (!selectedWallet) {
  selectedWallet = getSelectedWallet();
}
```

with default parameter values, ie.
```ts
selectedWallet = getSelectedWallet(),
selectedNetwork = getSelectedNetwork(),
```

### Type of change

- [x] Refactoring (improving code without creating new functionality)

### Tests

- [x] No test

### CODE REVIEW

File: src/utils/wallet/transactions.ts [hunk 1]
```diff
@@ -1228,21 +1206,17 @@ export const updateMessage = async ({
...
+	selectedWallet = getSelectedWallet(),
+	// eslint-disable-next-line @typescript-eslint/no-unused-vars
+	selectedNetwork = getSelectedNetwork(),
 }: {
 	message: string;
 	transaction?: ISendTransaction;
 	index?: number;
 	selectedWallet?: TWalletName;
 	selectedNetwork?: EAvailableNetwork;
 }): Promise<Result<string>> => {
```
Comment (coreyphillips) [inline src/utils/wallet/transactions.ts]:
```suggestion
}: {
	message: string;
	transaction?: ISendTransaction;
	index?: number;
}): Promise<Result<string>> => {
```
The `selectedWallet` & `selectedNetwork` variables are no longer needed in this method and can be safely removed.

File: src/utils/wallet/transactions.ts [hunk 2]
```diff
@@ -1621,18 +1579,14 @@ export const getFeeEstimates = async (
...
-	selectedWallet,
-	selectedNetwork,
+	// eslint-disable-next-line @typescript-eslint/no-unused-vars
+	selectedWallet = getSelectedWallet(),
+	// eslint-disable-next-line @typescript-eslint/no-unused-vars
+	selectedNetwork = getSelectedNetwork(),
 }: {
 	selectedWallet?: TWalletName;
 	selectedNetwork?: EAvailableNetwork;
 }): EFeeId => {
```
Comment (coreyphillips) [inline src/utils/wallet/transactions.ts]: The `selectedWallet` & `selectedNetwork` variables are no longer needed in the `getSelectedFeeId` method and can be safely removed.

**Review by coreyphillips — CHANGES_REQUESTED** [review]

File: src/utils/wallet/transactions.ts (hunk 2 above)
Comment (ovitrif) [reply src/utils/wallet/transactions.ts]: True dat! Thank you, it's now removed 🙇🏻

**Review by pwltr — APPROVED** [review]

**Review by coreyphillips — APPROVED** [review]

**Comment by ovitrif** [comment]
@limpbrains e2e-ios fails here, should I be concerned? the one on macmini succeeds 🤔 

It seems unrelated to the changes (IMHO), though I wonder what might cause the failure

From build logs these seemed to stand out:
```
[!] Error installing hermes-engine
```
and
```
Errno::EEXIST - File exists @ syserr_fail2_in - /Users/runner/Library/Caches/CocoaPods/Pods/External/hermes-engine/297ebba3ddadb20a86a625cc0bfe59ba-9180d/destroot/Library/Frameworks/universal/hermes.xcframework/ios-arm64_x86_64-maccatalyst/hermes.framework/Resources
```

**Comment by limpbrains** [comment]
Yes, it is unrelated. Broken cocoapods version

**Pull request merged**